frontend/libs/studio-components/src/components/StudioGridSelector/StudioGridSelector.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import classes from './StudioGridSelector.module.css';
+import cn from 'classnames';
+
+type StudioGridSelectorProps = {
+  disabled?: boolean;
+  sliderValue?: number;
+  handleSliderChange: (newValue: number) => void;
+};
```

Comment:
Maybe it would make more sense to extend `HTMLAttributes<HTMLInputElement>` and apply the props on the `<input>` element?

Reply:
@TomasEng hence my example implemented inputPros. Then the consumer can control both root and input element. 😊

Reply:
Nice, I didn't see that. On the other hand, this makes me think: Is this actually necessary? It does increase the flexibility of the component indeed, but also the complexity. And we are currently only planning to use it in one place. Wouldn't it be better to wait with adding this until we approach a use case where this is necessary? Let's have a talk about this after the holidays.

Reply:
It's a valid point that it increases complexity. The advantage of expanding and following patterns like inputProps is that it makes it a bit easier to adhere to the 'open for extension, but closed for modification' principle. 😊 Perhaps this component is unique enough that it might be unnecessary, but in a general sense, I am supportive of facilitating the extension of both the root element and the most central 'inner' element. Which is the `<input>` within this use-case.